### ISSUE TITLE
Train loss rises normally, validate loss has been fluctuating in a small range

### ISSUE THREAD

**Issue by zouxuelian** [bearpaw/pytorch-pose#56]
Hello, I have the following problems after training your lsp data set:
Epoch	LR	Train Loss	Val Loss	Train Acc	Val Acc	
1.000000	0.000250	0.007550	0.006845	0.050514	0.103110	
2.000000	0.000250	0.005672	0.028946	0.189198	0.218423	
3.000000	0.000250	0.005463	0.006511	0.327330	0.236230	
4.000000	0.000250	0.005265	0.010601	0.425851	0.225216	
5.000000	0.000250	0.005135	0.016247	0.476834	0.197104	
6.000000	0.000250	0.004991	0.010484	0.526583	0.235236	
7.000000	0.000250	0.004877	0.006420	0.563533	0.240461	
8.000000	0.000250	0.004769	0.008257	0.597851	0.228115	
9.000000	0.000250	0.004665	0.014668	0.626140	0.203795	
10.000000	0.000250	0.004563	0.013019	0.652237	0.212683	
11.000000	0.000250	0.004466	0.010125	0.678059	0.210265	
12.000000	0.000250	0.004363	0.011066	0.698233	0.238305	
13.000000	0.000250	0.004259	0.007450	0.719176	0.212096	
14.000000	0.000250	0.004150	0.007616	0.740357	0.254597	
15.000000	0.000250	0.004041	0.007984	0.757074	0.197692	
16.000000	0.000250	0.003930	0.007641	0.771822	0.249571	
17.000000	0.000250	0.003827	0.007281	0.784899	0.212632	
18.000000	0.000250	0.003716	0.007883	0.796875	0.213584	
19.000000	0.000250	0.003610	0.007357	0.808176	0.229985	
20.000000	0.000250	0.003501	0.006785	0.819266	0.226866	
21.000000	0.000250	0.003399	0.007181	0.827932	0.201103	
22.000000	0.000250	0.003286	0.007712	0.836886	0.211426	
23.000000	0.000250	0.003203	0.007294	0.842912	0.233875	
24.000000	0.000250	0.003100	0.008075	0.850344	0.209791	
25.000000	0.000250	0.003014	0.006982	0.857866	0.215311	
26.000000	0.000250	0.002944	0.006776	0.862607	0.228163	
27.000000	0.000250	0.002844	0.006760	0.869830	0.232245	
28.000000	0.000250	0.002769	0.007036	0.876174	0.231452	
29.000000	0.000250	0.002700	0.006785	0.880423	0.234402	
30.000000	0.000250	0.002629	0.007129	0.885608	0.237062	
31.000000	0.000250	0.002576	0.006889	0.890188	0.229911	
32.000000	0.000250	0.002514	0.006891	0.894307	0.207830	
33.000000	0.000250	0.002432	0.006683	0.900188	0.187527	
34.000000	0.000250	0.002380	0.006908	0.903378	0.189255	
35.000000	0.000250	0.002328	0.006710	0.908291	0.185184	
36.000000	0.000250	0.002277	0.006706	0.910652	0.209814	
37.000000	0.000250	0.002212	0.006474	0.915414	0.218596	
38.000000	0.000250	0.002175	0.006554	0.917380	0.213894	
39.000000	0.000250	0.002116	0.006439	0.921337	0.187475	
40.000000	0.000250	0.002074	0.006574	0.923499	0.208656	
41.000000	0.000250	0.002032	0.006839	0.925389	0.228495	
42.000000	0.000250	0.001996	0.006594	0.928320	0.194761	
43.000000	0.000250	0.001951	0.006584	0.929461	0.203907	
44.000000	0.000250	0.001918	0.006693	0.931813	0.220581	
45.000000	0.000250	0.001867	0.006530	0.934278	0.211442	
46.000000	0.000250	0.001824	0.006781	0.935595	0.223957	
47.000000	0.000250	0.001789	0.006597	0.936730	0.201136	
48.000000	0.000250	0.001754	0.006832	0.938524	0.187588	
49.000000	0.000250	0.001740	0.006638	0.939773	0.202755	
50.000000	0.000250	0.001678	0.006663	0.941929	0.212335	
51.000000	0.000250	0.001659	0.006536	0.943358	0.206499	
52.000000	0.000250	0.001622	0.006547	0.944517	0.194839	
53.000000	0.000250	0.001596	0.006524	0.945802	0.193153	
54.000000	0.000250	0.001569	0.006756	0.946932	0.195709	
55.000000	0.000250	0.001533	0.006639	0.948008	0.204389	
56.000000	0.000250	0.001507	0.006420	0.948236	0.186468	
57.000000	0.000250	0.001478	0.006393	0.949928	0.181633	
58.000000	0.000250	0.001449	0.006496	0.951115	0.195551	
59.000000	0.000250	0.001436	0.006596	0.951629	0.193789	
60.000000	0.000250	0.001406	0.006721	0.952892	0.214011	
61.000000	0.000025	0.001201	0.006459	0.956363	0.202763	
62.000000	0.000025	0.001113	0.006481	0.957531	0.205997	
63.000000	0.000025	0.001070	0.006464	0.958235	0.196485	
64.000000	0.000025	0.001040	0.006498	0.958765	0.194456	
65.000000	0.000025	0.001016	0.006437	0.959615	0.193237	
66.000000	0.000025	0.000998	0.006561	0.960101	0.198008	
67.000000	0.000025	0.000981	0.006560	0.960603	0.192700	
68.000000	0.000025	0.000967	0.006568	0.961013	0.192771	
69.000000	0.000025	0.000953	0.006591	0.961577	0.195998	
70.000000	0.000025	0.000942	0.006691	0.962032	0.196946	
71.000000	0.000025	0.000931	0.006613	0.962351	0.189600	
72.000000	0.000025	0.000919	0.006616	0.962809	0.185571	
73.000000	0.000025	0.000911	0.006642	0.963139	0.177612	
74.000000	0.000025	0.000901	0.006688	0.963402	0.192683	
75.000000	0.000025	0.000892	0.006641	0.963935	0.189106	
76.000000	0.000025	0.000883	0.006648	0.964253	0.183414	
77.000000	0.000025	0.000873	0.006751	0.964922	0.190603	
78.000000	0.000025	0.000866	0.006633	0.965025	0.175296	
79.000000	0.000025	0.000857	0.006689	0.965206	0.185154	
80.000000	0.000025	0.000850	0.006707	0.965530	0.182363	
81.000000	0.000025	0.000843	0.006735	0.966035	0.180616	
82.000000	0.000025	0.000835	0.006739	0.966482	0.181994	
83.000000	0.000025	0.000828	0.006807	0.966716	0.187205	
84.000000	0.000025	0.000822	0.006734	0.967067	0.185494	
85.000000	0.000025	0.000817	0.006645	0.967013	0.176170	
86.000000	0.000025	0.000811	0.006775	0.967535	0.184640	
87.000000	0.000025	0.000803	0.006733	0.967725	0.174413	
88.000000	0.000025	0.000797	0.006814	0.968161	0.178507	
89.000000	0.000025	0.000790	0.006726	0.968639	0.171258	
90.000000	0.000025	0.000787	0.006763	0.968690	0.174545

I don't know why
